.github/scripts/sync_labels.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+import sys
+import requests
+import json
+from github import Github
+
+try:
+    github_token = os.environ["GITHUB_TOKEN"]
+except KeyError:
+    print("Please set the 'GITHUB_TOKEN' environment variable")
+    sys.exit(1)
+
+
+def parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--repository', type=str, required=True,
+                        help='Github repository name (e.g., scylladb/scylladb)')
+    parser.add_argument('--number', type=int, required=True,
+                        help='Pull request number to sync labels from linked issue')
+    parser.add_argument('--is_issue', action='store_true', help='   Determined if label change is in Issue or not')
+    parser.add_argument('--label_action', type=str, choices=['opened', 'labeled', 'unlabeled'], required=True, help='Sync labels action')
+    return parser.parse_args()
+
+
+def get_pr_request_body(repo, number):
+    url = f"https://api.github.com/repos/{repo}/pulls/{number}"
+    headers = {
+        'Authorization': f'token {github_token}',
+        'Accept': 'application/vnd.github.v3+json'
+    }
+    response = requests.get(url, headers=headers)
+    if response.ok:
+        pr_details = response.json()
+        return pr_details['body']
```

Comment:
body? why not the labels? (btw, I think this is going to be a generic function, which fetches pull requests - I reckon we'll need more than one item from a PR

Reply:
We search in the PR body for linked issues with the `Fixes` prefix. 

And it's also working with multiple `Fixes` so in case a PR is fixing 2 issues, we will update both

Reply:
Limiting the function to body only `get_pr_request_body` is against OOP and will make future changes difficult,
So it's not about work/doesn't-work but going the pythonic way you can choose to go to or not